### PR TITLE
docu upgrade for stopPlaces

### LINKED ISSUE #512: Too generic type for stop places in SiteFrame
The pull request introducing new modes (https://github.com/NeTEx-CEN/NeTEx/commit/147c902f0df0eee1e727acdba9eed392581d0170)
modified the type of stop places in a site frame from` StopPlace `to the more generic `StopPlace_` :

https://github.com/NeTEx-CEN/NeTEx/blob/3d1d9e716ffd2577a58b0a7b6bf9770ed5bf3276/xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_version.xsd#L113

where:
`<xsd:element ref="StopPlace" maxOccurs="unbounded"/>`
became:
`<xsd:element ref="StopPlace_" maxOccurs="unbounded"/>`

As a result, type checking is relaxed and it is now possible to refer to other types in SiteFrame/stopPlaces, such as:
```

<SiteFrame>
    <stopPlaces>
        <Parking/>
        <PointOfInterest/>
    </stopPlaces>
</SiteFrame>
```

These types have their own containers (`<parkings>,` `<pointOfInterests>`, ...) and should not be mixed with `<stopPlaces>`

We believe that this change was not intended and should be reverted.

### PR DESCRIPTION
**Pull request by ue71603**
to fix #512

https://github.com/NeTEx-CEN/NeTEx/issues/512

### CODE REVIEW

**Review by skinkie — APPROVED** [review]
My honest reply: this should not have been put under "stopPlaces". But "stopPlaces" should have been renamed into places.

**Comment by ue71603** [comment]
@skinkie  I can rename it. But I think we need broad approval there @Aurige @nick-knowles

**Comment by ue71603** [comment]
Details for longer term solution: https://github.com/NeTEx-CEN/NeTEx/issues/609
Site in

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_version.xsd [hunk 1]
```diff
@@ -105,7 +105,7 @@ Rail transport, Roads and Road transport
 	<!-- ===ENTIITY IN VERSION IN FRAME====(Used in SITE FRAME)=================================================== -->
 	<xsd:complexType name="stopPlacesInFrame_RelStructure">
 		<xsd:annotation>
-			<xsd:documentation>Type for containment in frame of STOP PLACE.</xsd:documentation>
+			<xsd:documentation>Type for containment in frame of STOP PLACE, VEHICLE MEETING POINTs, TAXI RANKs, POINT OF INTERESTs, PARKINGs and the like.</xsd:documentation>
```
Comment (ue71603) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_version.xsd:108]:
```suggestion
			<xsd:documentation>Type for containment in frame of STOP PLACE, VEHICLE MEETING POINT, TAXI RANK.</xsd:documentation>
```

**Review by Aurige — APPROVED** [review]

**Comment by ue71603** [comment]
Do it also in other places where StopPlace_ is mentioned.

**Pull request merged**